pkg/odo/cli/component/create.go

```diff
@@ -463,7 +439,7 @@ func NewCmdCreate(name, fullName string) *cobra.Command {
 	componentCreateCmd.Flags().StringVarP(&co.componentBinary, "binary", "b", "", "Use a binary as the source file for the component")
 	componentCreateCmd.Flags().StringVarP(&co.componentGit, "git", "g", "", "Use a git repository as the source file for the component")
 	componentCreateCmd.Flags().StringVarP(&co.componentGitRef, "ref", "r", "", "Use a specific ref e.g. commit, branch or tag of the git repository")
-	componentCreateCmd.Flags().StringVarP(&co.componentLocal, "local", "l", "", "Use local directory as a source file for the component")
+	componentCreateCmd.Flags().StringVar(&co.componentContext, "context", "", "Use local directory as a source file for the component")
```

Comment:
perhaps I missed a discussion, but why is `--local` removed and now it's context, but the description still says `local`?

Reply:
context now serves the dual purpose of serving as the path to config file and config file is expected to reside closer to the source so no need to have local flag anymore

And fixed the command flag explanation accordingly